### DISCUSSION TITLE
How can I open a local folder?

### DISCUSSION THREAD

**Discussion by lee12180** [coder/code-server#4466] [Q&A] [answered]
In the case of files, can see the "show local" button when "open file", but the folder does not see this button.

![캡처](https://user-images.githubusercontent.com/16605694/140720670-bf218a0e-a7cf-4a20-8cdc-a8f600e8764a.PNG)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Maybe can you try opening the home directory and moving your specified folder up there?

**Reply by jsjoeio** [reply]
Let us know if this works!

**Reply by lee12180** [reply]
I installed ubuntu and code-server using wsl2 in window 10.

After accessing localhost:8080 on Windows 10,

The ubuntu directory is normally accessible,
There is no way to open the folder of window10. (Single file is possible)

Is the question stupid? I'm sorry for my poor English.

**Reply by lg-nineoneninethreenine** [reply]
Until you upload your folder, it won't work.

**Reply by lg-nineoneninethreenine** [reply]
Oh, wait a minute! Can you try opening this folder: `/mnt/c/Users/user`? Replace `user` with your username, it might work.

**Reply by lee12180** [reply]
Yes
I can't select a folder with File Explorer pop-up, but it's possible to select a folder by entering a path.
Thank you.
